app/controllers/comments_controller.rb

```diff
@@ -1,5 +1,14 @@
 class CommentsController < ApplicationController
-  def index; end
+  def new
+    @comments = Comment.new
+  end
 
-  def show; end
+  def create
+    new_comment = Comment.new(Text: params[:comment][:Text], post: Post.find(params[:post_id]), author: current_user)
+    if new_comment.save
+      redirect_to user_posts_path
+    else
+      render :new
+    end
+  end
```

Comment:
- [ ] I can see you have added a new comment route but currently I am not able to access it because in the first place a post is not added that means you can't add a comment without a post please address it